From: Kent Hou Man <knthmn0@gmail.com>

This laptop also defines its active-low keyboard IRQ in legacy format.
Add IRQ override skipping for it.

Cc: <stable@vger.kernel.org>
Signed-off-by: Kent Hou Man <knthmn0@gmail.com>
Signed-off-by: Chuanhong Guo <gch981213@gmail.com>
---
Change since v2: none

 drivers/acpi/resource.c | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/drivers/acpi/resource.c b/drivers/acpi/resource.c
index 30c0d85b2bf2..eff615f51d07 100644
--- a/drivers/acpi/resource.c
+++ b/drivers/acpi/resource.c
@@ -400,6 +400,13 @@ static const struct dmi_system_id medion_laptop[] = {
 };
 
 static const struct dmi_system_id irq1_edge_low_shared[] = {
+	{
+		.ident = "Asus Zenbook S 13 OLED UM5302",
+		.matches = {
+			DMI_MATCH(DMI_SYS_VENDOR, "ASUSTeK COMPUTER INC."),
+			DMI_MATCH(DMI_BOARD_NAME, "UM5302TA"),
+		},
+	},
 	{
 		.ident = "Lenovo ThinkBook 14 G4+ ARA",
 		.matches = {
-- 
2.36.1
